src/constants.tsx

```diff
@@ -88,6 +88,7 @@ export const OPENSRP_ORG_PRACTITIONER_ENDPOINT = 'organization/practitioner';
 export const OPENSRP_DEL_PRACTITIONER_ROLE_ENDPOINT = 'practitionerRole/deleteByPractitioner';
 export const OPENSRP_ADD_PRACTITIONER_ROLE_ENDPOINT = 'practitionerRole/add';
 export const OPENSRP_USERS_ENDPOINT = 'user';
+export const OPENSRP_FIND_EVENTS_ENDPOINT = 'event/findById';
```

Comment:
Is this in the `.env.sample`?

Reply:
@cKellyDesign Doesn't seem like we have been placing the endpoints in env.sample.

Reply:
If there should be there then maybe i can create a ticket to move them there

Reply:
I think @cKellyDesign agrees with you that we don't need to do that :)

Reply:
oh sorry, for some reason i thought this was in `env.ts`, my bad!